process_report/tests/unit_tests.py

```diff
@@ -175,3 +176,65 @@ def test_export_pi(self):
         self.assertNotIn('ProjectA', pi_df['Project - Allocation'].tolist())
         self.assertNotIn('ProjectB', pi_df['Project - Allocation'].tolist())
         self.assertNotIn('ProjectC', pi_df['Project - Allocation'].tolist())
+
+
+class TestGetInstitute(TestCase):
+
+    def setUp(self):
+
+        data = {
+            'Manager (PI)': ['quanmp@bu.edu', 'c@mclean.harvard.edu', 'b@harvard.edu', 'fake', 'pi@northeastern.edu'],
+            'Project - Allocation': ['ProjectA', 'ProjectB', 'ProjectC', 'ProjectD', 'ProjectE'],
+            'Answer': ["Boston University", "McLean Hospital", "Harvard University", "", "Northeastern University"]
+        }
+        self.data = pandas.DataFrame(data)
+
+    def test_get_pi_institution(self):
+        for i, row in self.data.iterrows():
+            institution_name = process_report.get_institution_from_pi(row['Manager (PI)'])    
+            self.assertEqual(institution_name, row['Answer'])
+
+@skipIf(process_report.apply_credits_0002 not in process_report.applied_credits, "Skipping test for credit 0002 because credit not enabled")
+class TestCredit0002(TestCase):
+    def setUp(self):
+
+        data = {
+            'Manager (PI)': ['PI1', 'PI1', 'PI2', 'PI3', 'PI4', 'PI4'],
+            'Project - Allocation': ['ProjectA', 'ProjectB', 'ProjectC', 'ProjectD', 'ProjectE', 'ProjectF'],
+            'Cost': [10, 100, 10000, 5000, 800, 1000]
```

Comment:
is there a reason you did not include and answer key here?

Reply:
@QuanMPhm The assertions made here check to see if the correct credit and balances were given to the PIs. I could have put the answers for the credits and balances in that dict, but that would just require more code to reference those answers in the test function `test_apply_credit_0002()`.